### PR TITLE
AV-207315-master: Support NSX-T in shared vip L4 services

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

**Comment by akshayhavile** [comment]
build ako

File: internal/nodes/avi_model_advl4_translator.go [hunk 1]
```diff
@@ -551,9 +551,17 @@ func (o *AviObjectGraph) ConstructSharedVipSvcLBNode(sharedVipKey, namespace, ke
 		vsVipNode.BGPPeerLabels = lib.GetGlobalBgpPeerLabels()
 	}
 
+	t1lr := lib.GetT1LRPath()
+	if t1lr != "" {
```
Comment (abhishekbsingh) [inline internal/nodes/avi_model_advl4_translator.go:555]: If it's empty and T1lr is defined in the AviInfraSetting, vrfcontext will not be set to empty string. We should also check if t1lr is defined in the aviinfrasetting cr and based on that set the vrfcontext and t1lr fields accordingly.

Comment (abhishekbsingh) [reply internal/nodes/avi_model_advl4_translator.go:555]: oh, I see you have updated the buildWithInfraSetting function, please ignore the above comment

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Review by DixitAakash — APPROVED** [review]
LGTM

**Review by abhishekbsingh — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by pkoshtavmware** [comment]
build ako

**Comment by pkoshtavmware** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Pull request merged**